simplq/src/components/common/Footer.jsx

```diff
@@ -44,24 +49,34 @@ export default () => (
         <div className={styles['list-container']}>
           <ul>
             <li>
-              <a href="https://iimb.qualtrics.com/jfe/form/SV_aY8FY91ztRX9NvD">Contact Us</a>
+              <a href="https://iimb.qualtrics.com/jfe/form/SV_aY8FY91ztRX9NvD">
+                <PhoneIcon fontSize={"large"} />
+              </a>
             </li>
             <li>
-              <a href="https://github.com/SimplQ/simplQ-frontend">Github</a>
+              <a href="https://github.com/SimplQ/simplQ-frontend">
+                <GitHubIcon fontSize={"large"} />
+              </a>
             </li>
             <li>
               <a href="https://medium.com/@raimazach/virtualizing-queues-a-long-overdue-alternative-to-physical-queues-bfdc4b51070f">
-                Medium
+               <img src="images/ICON-MEDIUM.jpg" />
               </a>
             </li>
             <li>
-              <a href="https://www.youtube.com/channel/UCAb9PSXvrGZ4vvSneK1Nrow">Youtube</a>
+              <a href="https://www.youtube.com/channel/UCAb9PSXvrGZ4vvSneK1Nrow">
+                <YouTubeIcon className={styles['youtube-icon']} fontSize={"large"}/>
```

Comment:
Why did we have to add this extra class just for youtube icon?

Reply:
Bcz that's youtube icon looks way too small than others in large size. So In order to maintain size, I have used bit CSS there.

Reply:
Got it. Can you add a small one line comment here:
```
{/* Youtube Icon was looking smaller that others, setting heigh to match other icons */}
```

Reply:
@daltonfury42 Done!

Reply:
Thaaanks